Guide updates

moschel opened the pull request:
Adding screenshots and doing a pass of editing the writing.

moschel committed:
adding images and small language changes to quick start guide

moschel committed:
more images

moschel committed:
updates to guide

moschel committed:
more screenshots

moschel committed:
fixing links

justinbmeyer commented:
It's not a page param, it's a page property. 

Sent from my iPhone

> On Oct 29, 2015, at 10:32 PM, Brian Moschel notifications@github.com wrote:
> 
> Adding screenshots and doing a pass of editing the writing.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/donejs/donejs/pull/394
> 
> Commit Summary
> 
> adding images and small language changes to quick start guide
> more images
> updates to guide
> more screenshots
> fixing links
> File Changes
> 
> D docs/about2.md (76)
> M docs/features.md (4)
> M docs/guide.md (199)
> A docs/theme/static/img/donejs-bootstrap.png (0)
> A docs/theme/static/img/donejs-chat1.png (0)
> A docs/theme/static/img/donejs-chat2.png (0)
> A docs/theme/static/img/donejs-chat3.png (0)
> A docs/theme/static/img/donejs-deploy.png (0)
> A docs/theme/static/img/donejs-firebase.png (0)
> A docs/theme/static/img/donejs-generator.png (0)
> A docs/theme/static/img/donejs-helloworld.png (0)
> A docs/theme/static/img/donejs-init.png (0)
> A docs/theme/static/img/donejs-ios.png (0)
> A docs/theme/static/img/donejs-localstorage.png (0)
> A docs/theme/static/img/donejs-model-generator.png (0)
> A docs/theme/static/img/donejs-nw1.png (0)
> A docs/theme/static/img/donejs-nw2.png (0)
> A docs/theme/static/img/donejs-prodmode.png (0)
> A docs/theme/static/img/donejs-tabs.png (0)
> A docs/theme/static/img/donejs-twobrowsers.png (0)
> A docs/theme/static/img/donejs-viewsource.png (0)
> Patch Links:
> 
> https://github.com/donejs/donejs/pull/394.patch
> https://github.com/donejs/donejs/pull/394.diff
> —
> Reply to this email directly or view it on GitHub.

justinbmeyer commented:
Why add have/will/etc instead of using just the verb. 

We create vs we will create
We created vs we have created

I don't think the modifiers add anything, but make the actions seem a little more remote /distant. 

Sent from my iPhone

> On Oct 29, 2015, at 10:32 PM, Brian Moschel notifications@github.com wrote:
> 
> Adding screenshots and doing a pass of editing the writing.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/donejs/donejs/pull/394
> 
> Commit Summary
> 
> adding images and small language changes to quick start guide
> more images
> updates to guide
> more screenshots
> fixing links
> File Changes
> 
> D docs/about2.md (76)
> M docs/features.md (4)
> M docs/guide.md (199)
> A docs/theme/static/img/donejs-bootstrap.png (0)
> A docs/theme/static/img/donejs-chat1.png (0)
> A docs/theme/static/img/donejs-chat2.png (0)
> A docs/theme/static/img/donejs-chat3.png (0)
> A docs/theme/static/img/donejs-deploy.png (0)
> A docs/theme/static/img/donejs-firebase.png (0)
> A docs/theme/static/img/donejs-generator.png (0)
> A docs/theme/static/img/donejs-helloworld.png (0)
> A docs/theme/static/img/donejs-init.png (0)
> A docs/theme/static/img/donejs-ios.png (0)
> A docs/theme/static/img/donejs-localstorage.png (0)
> A docs/theme/static/img/donejs-model-generator.png (0)
> A docs/theme/static/img/donejs-nw1.png (0)
> A docs/theme/static/img/donejs-nw2.png (0)
> A docs/theme/static/img/donejs-prodmode.png (0)
> A docs/theme/static/img/donejs-tabs.png (0)
> A docs/theme/static/img/donejs-twobrowsers.png (0)
> A docs/theme/static/img/donejs-viewsource.png (0)
> Patch Links:
> 
> https://github.com/donejs/donejs/pull/394.patch
> https://github.com/donejs/donejs/pull/394.diff
> —
> Reply to this email directly or view it on GitHub.

moschel commented:
I added "will" in a few places for consistency. 90% of the intro sections use that verb tense, so I added where it was missing. Without it, it reads weird.
